src/config.rs

```diff
@@ -71,6 +93,44 @@ fn load_file_contents(path: path::PathBuf) -> Result<String, Error> {
     Ok(contents)
 }
 
+fn extract_curly_brace(line: &str) -> Vec<String> {
+    let mut before_curly_brace = String::new();
+    let mut after_curly_brace = String::new();
+    let mut start: usize = usize::MAX;
+    let mut end: usize = usize::MAX;
+    let mut output: Vec<String> = Vec::new();
+    for (i, c) in line.chars().enumerate() {
+        if c == '{' {
+            before_curly_brace = line[..i].to_string();
+            start = i;
+            continue;
+        }
+        if c == '}' {
+            after_curly_brace = line[i + 1..].to_string();
+            end = i;
+        }
+    }
+    if start >= end {
+        return vec![line.to_string()];
+    }
+    for item in line[start + 1..end].split(',') {
+        if item.contains('-') {
+            let mut range = item.split('-').map(|s| s.trim());
+            let begin = range.next().unwrap().parse::<char>().unwrap() as u8;
```

Comment:
Here we use two `unwrap` but can't handle `None` and `Err`

Reply:
If there is a `None` or `Err`, it should cancel the curly brace extraction, right?

Reply:
If yes, we can return a normal extraction and let `parse_keybind` handle the error output